crm_phonecall_summary_predefined/README.rst

```diff
@@ -0,0 +1,66 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==================================
+Restricted Summary for Phone Calls
+==================================
+
+This module extends the functionality of CRM phone calls to support setting a
+limited list of possible phone call summaries, that only the sales manager can
+edit.
+
+Configuration
+=============
+
+To configure the possible summary options, go to:
+
+* Go to *Sales > Configuration > Phone Calls > Summaries*.
+
```

Comment:
*  Go to *Sales > Configuration > Sales & Opportunities > Phone calls > Summaries*.